### ISSUE TITLE
Qt5 builds failing on Bitrise — missing /usr/local/.//mkspecs/macx-clang

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-gl-native#6712]
Qt5 macOS builds have been failing on Bitrise for the past few days, as seen in https://www.bitrise.io/build/243eaa19e4ba6066 (#6601) and https://www.bitrise.io/build/bda5e310bc1dedf1 (#6709):

```
-- [Mason] Unpacking package to mason_packages/headers/geojson/0.3.2...
CMake Error at /usr/local/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:15 (message):
  The imported target "Qt5::Core" references the file

     "/usr/local/.//mkspecs/macx-clang"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/usr/local/lib/cmake/Qt5Core/Qt5CoreConfigExtras.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /usr/local/lib/cmake/Qt5Core/Qt5CoreConfigExtras.cmake:50 (_qt5_Core_check_file_exists)
  /usr/local/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:141 (include)
  platform/qt/qt5.cmake:1 (find_package)
  platform/qt/qt.cmake:72 (include)
  platform/qt/config.cmake:1 (include)
  CMakeLists.txt:51 (include)


-- Configuring incomplete, errors occurred!
See also "/Users/vagrant/git/build/qt-macos-x86_64/Debug/CMakeFiles/CMakeOutput.log".
make: *** [build/qt-macos-x86_64/Debug/build.ninja] Error 1
```

/cc @brunoabinader @tmpsantos

**Comment by tmpsantos** [comment]
Is it possible that the Qt version has changed and now brew is fetching a
broken version?

**Comment by brunoabinader** [comment]
> Is it possible that the Qt version has changed and now brew is fetching a broken version?

Yes. Homebrew is now using Qt 5.7.0 - while our Bitrise script is [creating symlinks for Qt 5.6.1.1](https://github.com/mapbox/mapbox-gl-native/blob/7dbc302b2f048902124e8f3d9d368c14204cfae5/platform/qt/bitrise-qt5.yml#L37-L38). Fixed in #6716.
